duckduckgo/duckduckgo-privacy-extension
# Expand clickable area that leads to subviews

**Reviewer:** @laurengarcia 



## Description:
Header, tracker networks, privacy practices, and all trackers lines should now be entirely clickable.
Previously only clicking on the arrow or the link had any effect.
Also changed the cursor in those areas to indicate they're clickable.


## Steps to test this PR:
1. Build and reload extension
2. Visit any site
3. All the elements listed in the description should be clickable, showing the cursor pointer
4. Popup switches correctly between main view and subviews


## Automated tests:

###### Reviewer Checklist:


###### PR Author Checklist:
